nodejs/awsx/lb/loadBalancer.ts

```diff
@@ -100,6 +114,15 @@ export interface LoadBalancerArgs {
      */
     external?: boolean;
 
+    /**
+     * Allows an existing load balancer to be used. There are cases where users want to use the
+     * convenience of awsx, but already have an existing load balancer either from another stack
+     * or already in their account.
+     *
+     * If this value is set, no other arguments are allowed.
+     */
+    loadBalancer?: aws.lb.LoadBalancer;
```

Comment:
Note that the concrete subclasses don't actually have args interfaces that build on the `LoadBalancerArgs` interface, so the `NetworkLoadBalancerArgs` and `ApplicationLoadBalancerArgs` would actually need a `loadBalancer` field as well.